source/_integrations/weblink.markdown

```diff
@@ -11,7 +11,7 @@ ha_qa_scale: internal
 The `weblink` integration allows you to display links in the Home Assistant frontend.
 
 <div class='note'>
-The below documentation applies to the classic "States" user interface. Starting with Home Assistant 0.86, Lovelace is the new default interface. For information on configuring weblinks in Lovelace please follow [these instructions](/lovelace/entities/#weblink) instead.
+The below documentation applies to the classic "States" user interface. Starting with Home Assistant 0.86, Lovelace is the new default interface. For information on configuring weblinks in Lovelace please follow <a href="/lovelace/entities/#weblink">these instructions</a> instead.
```

Comment:
This is not how we handle this. Please restore the Markdown and add an empty line after the opening and just before closing the `div`. That allows the parser to switch context.